rest-model/src/main/java/org/jboss/pnc/rest/utils/JsonOutputConverterMapper.java

```diff
@@ -47,6 +50,7 @@ public static String apply(Object objectToBeConverted) {
             try {
                 return mapper.writeValueAsString(objectToBeConverted);
             } catch (JsonProcessingException e) { //TODO remove exception masking
+                log.warn("Could not convert object to JSON", e);
```

Comment:
@Matej: What do you think about logging the object here too?
The only cons that I can think of here is that we can have some sensitive data that might end up in the log in this case. What is your opinion?